src/v2/view-builder/utils/FactorUtil.js

```diff
@@ -14,45 +14,39 @@ import { getPasswordComplexityDescriptionForHtmlList } from '../../../util/Facto
 
 const factorData = {
   'email': {
-    label: loc('oie.email', 'login'),
-    description: '',
+    description: loc('oie.email.authenticator.description', 'login'),
     iconClassName: 'mfa-okta-email',
   },
 
   'password': {
-    label: loc('oie.password', 'login'),
-    description: '',
+    description: loc('oie.password.authenticator.description', 'login'),
     iconClassName: 'mfa-okta-password',
   },
 
   'phone': {
-    label: loc('oie.phone', 'login'),
-    description: '',
+    description: loc('oie.phone.authenticator.description', 'login'),
     iconClassName: 'mfa-okta-phone',
   },
 
   'security_question': {
-    label: loc('oie.security.question', 'login'),
-    description: '',
+    description: loc('oie.security.question.authenticator.description', 'login'),
     iconClassName: 'mfa-okta-security-question',
   },
 
   // Will get rid of this after fully implementing verify. OKTA-301557
   'webauthn': {
-    label: loc('oie.webauthn', 'login'),
     description: loc('oie.webauthn.description', 'login'),
     iconClassName: 'mfa-webauthn',
   },
 
   'security_key': {
-    label: loc('oie.webauthn', 'login'),
     description: loc('oie.webauthn.description', 'login'),
     iconClassName: 'mfa-webauthn',
   },
 
   'app': {
-    label: loc('oie.app', 'login'),
-    description: '',
+    label: loc('oie.authenticator.okta_verify.label', 'login'),
```

Comment:
Update the mapping in i18nTransformer then the `label` can be removed.

Reply:
Spoke with @magizh-okta -
* Okta Verify label comes from backend.
    - Removed the entry (`oie.app`) from `login.properties`.
    - Added a key (`oie.authenticator.app.label`) in `i18nTransformer` like other authenticators.

* Webauthn label comes from `login.properties`. Updated login.properties with a key (`oie.authenticator.webauthn.label`) and updated `i18nTransformer`.